Dockerfile

```diff
@@ -0,0 +1,46 @@
+# syntax = docker/dockerfile:1.4
+
+# BUILD FRONTEND STAGE
+FROM node:16-alpine as frontend-builder
+WORKDIR /app
+
+# Install Make
+RUN apk add --no-cache make
+
+# Install Yarn v3
+RUN corepack enable
+RUN corepack prepare yarn@stable --activate
+RUN yarn set version stable
+
+# Copy and install deps
+COPY --link Makefile .
+COPY --link web web
+RUN make web/install-deps
+
+# Copy all file
+COPY --link . .
+
+# Build frontend
+RUN make web/build
+
+# BUILD BACKEND STAGE
+FROM golang:1.18 as backend-builder
+WORKDIR /app
+COPY --link go.mod go.sum ./
+RUN go mod download
+COPY --link . .
+COPY --link --from=frontend-builder /app/web/dist /app/web/dist
+RUN make bin
+
+# RUN STAGE
+FROM alpine:latest
```

Comment:
I would avoid using latest tag and might adopt something like distroless base image. But I would like project maintainers to make this decision

Reply:
@inductor I think distroless better than alpine, smaller and only necessary dependency.

Reply:
I updated alpine to ver 3.17

Reply:
@inductor Do you know https://www.chainguard.dev/chainguard-images?